docs/src/content/docs/pt-pt/resources/community-content.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Conteúdo da Comunidade
+description: Descubra guias produzidos pela comunidade, artigos e vídeos para ajudar a aprender e construir com o Starlight!
+---
+
+:::tip[Inclua o seu!]
+Já produziu conteúdo sobre Starlight?
+Então abra um PR e adicione o seu link nesta página!
+:::
+
+import { CardGrid, LinkCard } from '@astrojs/starlight/components';
+
+## Artigos e Avaliações
+
+Aqui está uma coleção de posts e artigos para aprender mais sobre Starlight e sobre as experiências de outras pessoas:
+
+<CardGrid>
+	<LinkCard
+		href="https://devm.io/open-source/starlight-astro"
+		title="Geração de Site Estático com Starlight"
+		description="'Nenhuma ideia é tão grande ou pequena ao projetar componentes' — uma entrevista com Chris Swithinbank, Líder Starlight"
+	/>
+	<LinkCard
+		href="https://frontendatscale.com/blog/hybrid-frontend-architecture/"
+		title="Arquitetura Frontend Híbrida com Astro e Starlight"
+		description="Maxi Ferreira e Ben Holmes constroem um site de documentação com Starlight, TinaCMS, e um Playground de API interativo com autenticação."
+	/>
+	<LinkCard
+		href="https://www.olets.dev/posts/comparing-docs-site-builders-vuepress-vs-starlight/"
+		title="Comparando construtores de site de documentação: VuePress vs Starlight"
+		description="Como estes dois frameworks se comparam?"
+	/>
+</CardGrid>
+
+## Receitas e Guias
+
+Receitas são tipicamente guias passo a passo, curtos e focados, que levam o leitor a completar um exemplo funcional de uma tarefa específica.
+
+As receitas são também uma ótima maneira de adicionar novas funcionalidades ou comportamentos para seu projeto Starlight bastando seguir simples instruções passo a passo!
+
+Outros guias podem explicar conceitos relacionados a determinada área de conteúdo, como por exemplo usar imagens ou trabalhar com MDX.
```

Comment:
Small detail but in the original version these three lines are all together in one paragraph:
```suggestion
Receitas são tipicamente guias passo a passo, curtos e focados, que levam o leitor a completar um exemplo funcional de uma tarefa específica.
As receitas são também uma ótima maneira de adicionar novas funcionalidades ou comportamentos para seu projeto Starlight bastando seguir simples instruções passo a passo!
Outros guias podem explicar conceitos relacionados a determinada área de conteúdo, como por exemplo usar imagens ou trabalhar com MDX.
```